File: CrawlerBlog/Domain/Entities/Post.cs
using System;
using System.Collections.Generic;
using System.ComponentModel.DataAnnotations.Schema;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace Domain.Entities
{
    [Table("Post")]
    public class Post
    {
        public string id { get; set; }
        public string uri { get; set; }
        public string title { get; set; }
        public string cmtId { get; set; }
        public string summary { get; set; }
        public string totalComments { get; set; }
        public List<Comment> comments { get; set; }
    }
}

File: SPA/src/app/model/dto/Post.ts
import { Comment } from './Comment';

export interface Post {
  id: string;
  uri: string;
  title: string;
  summary: string;
  totalComments: string;
  comments: Array<Comment>;
}

export interface GetPostsDto {
  posts: Post[];
  totalPages: number;
}

export interface ChangeListDto {
  id: string;
  title: string;
  uri: string;
  commentsUpdated: string;
}
